docs/_docs/thin-clients/dotnet-thin-client.adoc

```diff
@@ -235,6 +235,20 @@ task named `org.foo.bar.AddOneTask` passing `1` as an input parameter:
 include::{sourceCodeFile}[tag=client-compute-task,indent=0]
 -------------------------------------------------------------------------------
 
+== Executing Ignite Services
+
+You can use the `IServicesClient` API to invoke an link:services/services[Ignite Service] that
+is *already deployed* in the cluster.
+
+The example below shows how to invoke the service named `MyService`:
+[source, csharp]
+-------------------------------------------------------------------------------
+include::{sourceCodeFile}[tag=client-services,indent=0]
+-------------------------------------------------------------------------------
+
+The deployed service can be implemented using Java or .NET.
```

Comment:
I would say "The thin client allows to execute services implemented in Java or C#"

Reply:
Fixed